### DISCUSSION TITLE
Is there a list of all "device types" somewhere?

### DISCUSSION THREAD

**Discussion by chansearrington** [dgreif/ring#695] [Q&A] [answered]
I'm trying to setup my system to hide certain device types. But I don't know what to call them.

I looked in the read.me and home page, but don't see a list anywhere. 

For example, I want my instance to be only doorbells, flood light pro camera, and spotlight camera, but what are the names of these device types?

And how do I write it in the config file?

`{
    "accessories": [],
    "platforms": [
        {
            "platform": "Ring",
            "refreshToken": "hidden",
            "onlyDeviceTypes": "camera, doorbell",
            "hideLightGroups": true,
            "hideDoorbellSwitch": true,
            "hideCameraMotionSensor": true,
            "hideCameraSirenSwitch": true,
            "hideInHomeDoorbellSwitch": true,
            "hideAlarmSirenSwitch": true,
            "hideUnsupportedServices": true,
            "sendCameraMotionNotificationsToTv": false,
            "sendDoorbellMotionNotificationsToTv": true,
            "beamDurationSeconds": 3600,
            "debug": true
        }
    ]
}`

**Comment by dgreif** [comment, marked as answer]
You can find all the device and camera types in https://github.com/dgreif/ring/blob/master/api/ring-types.ts

**Comment by rrman** [comment]
Anyone know where I can get the list of valid device types to use for the onlyDeviceTypes? The link mentioned in the July 28, 2021 post no longer works.

**Reply by smockle** [reply]
I think it’s the right side of this list: https://github.com/dgreif/ring/blob/41a9e15437fb68210c82526eb034237b0d1e274a/packages/ring-client-api/ring-types.ts#L4-L48

So, for example, `sensor.contact`.

**Reply by rrman** [reply]
Thanks!
